src/templates/partials/head.mjml

```diff
@@ -117,6 +119,12 @@
     padding-left: 16px;
   }
 
+  .article-recommended {
+    display: flex;
```

Comment:
please don't use flex layout since the email system is pretty [legacy](https://www.caniemail.com/search/?s=flex) and lots of CSS features are supported.

Reply:
try [mjml's tags](https://documentation.mjml.io/#basic-layout-example), if no workaround, please confirm with designer and make a UI adjustment.

Reply:
Already confirmed with designer and made the changes.